src/nxt_controller.c

```diff
@@ -1931,10 +1931,27 @@ nxt_controller_cert_in_use(nxt_str_t *name)
                 continue;
             }
 
-            nxt_conf_get_string(value, &str);
+            if (nxt_conf_type(value) == NXT_CONF_ARRAY) {
+                n = nxt_conf_array_elements_count(value);
 
-            if (nxt_strstr_eq(&str, name)) {
-                return 1;
+                for (i = 0; i < n; i++) {
```

Comment:
Similarly, we could now take advantage of C99 for loop initialisers and reduce the scope of `i` with

```c
for (uint32_t i = 0; i < n; i++) {
```